enterprise/internal/campaigns/webhooks.go

```diff
@@ -881,7 +896,7 @@ func (h *BitbucketServerWebhook) parseEvent(r *http.Request) (interface{}, *repo
 
 	sig := r.Header.Get("X-Hub-Signature")
 
-	rawID := r.FormValue(externalServiceIDParam)
+	rawID := r.FormValue(extsvc.ExternalServiceIDParam)
```

Comment:
Reading it in this context, I think adding "Webhook" to the name might make things clearer:

```suggestion
	rawID := r.FormValue(extsvc.WebhookParam)
```

Reply:
It's specific to the ID, so I think as it's in the extsvc package already I'll call it `extsvc.IDParam`